From 7ae986447a95f67c025578bc6b5f34f7d585f294 Mon Sep 17 00:00:00 2001
From: Steve Wainstead <wainstead@gmail.com>
Date: Sun, 7 Jun 2009 20:48:59 +0000
Subject: [PATCH 007/223] Changed (void *) to (int *) to satisfy longjmp();
 note that cb->jmp is of type jmp_buf, defined in
 setjmp.h.

---
 server/exceptions.c |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/server/exceptions.c b/server/exceptions.c
index 208c2e8..651511a 100644
--- a/server/exceptions.c
+++ b/server/exceptions.c
@@ -56,11 +56,11 @@ ES_RaiseException(Exception * exception, int value)
     if (!xb)
 	panic("Unhandled exception!");
 
-    for (cb = ES_exceptionStack; cb != xb && !cb->finally; cb = cb->link);
+    for (cb = ES_exceptionStack; cb != xb && !cb->_finally; cb = cb->link);
     ES_exceptionStack = cb;
     cb->id = exception;
     cb->value = value;
-    longjmp((void *) cb->jmp, 1);
+    longjmp((int *) cb->jmp, 1);
 }
 
 char rcsid_exceptions[] = "$Id: exceptions.c,v 1.3 1998-12-14 13:17:48 nop Exp $";
-- 
1.7.9.5

